src/test/java/org/apache/sling/resourceresolver/impl/mapping/MapEntriesTest.java

```diff
@@ -116,9 +116,9 @@ public void setup() throws Exception {
                 Collections.<Resource> emptySet().iterator());
         //when(resourceResolverFactory.getAliasPath()).thenReturn(Arrays.asList("/child"));
 
-        CopyOnWriteArrayList<String> aliasPath = new CopyOnWriteArrayList<>();
+        Set<String> aliasPath = new TreeSet<>();
         aliasPath.add("/parent");
-        for(int i = 1;i<testSize;i++){
+              for(int i = 1;i<testSize;i++){
```

Comment:
Indentation looks wrong. Also the Sling code-base normally uses white-space after the semicolons and around the comparator.